docs/changes/io.ascii/16930.feature.rst

```diff
@@ -0,0 +1,2 @@
+Reading ascii tables from files specified as `pathlib.Path` objects is now
+supported.
```

Comment:
This change is much narrower than suggested in the original text. `io.ascii.read()` already fully supports a `Path` object since the input is always passed through `_expand_user_if_path` in `ui.py`.
```suggestion
Add support for `pathlib.Path` objects in `io.ascii.core.BaseInputter.get_lines`.
```